pcdscalc/be_lens_calcs.py

```diff
@@ -0,0 +1,830 @@
+'''
+Module for Beryllium Lens Calculations
+'''
+import logging
+import os
+import shutil
+from datetime import date
+from itertools import product
+
+import numpy as np
+import xraydb as xdb
+
+
+logger = logging.getLogger(__name__)
+
+# Set of Be lenses with thicknesses:
+LENS_RADII = [50e-6, 100e-6, 200e-6, 300e-6, 500e-6, 1000e-6, 1500e-6]
+
+FWHM_SIGMA_CONVERSION = 2.35482004503
+WAVELENGTH_PHOTON = 1.2398
+
+
+def photon_to_wavelength(energy):
+    '''
+    Find the wavelength in micrometers, using the
+    photon energy in electronvolts.
+    The equation is approximately:
+    λ[µm] = 1.2398 / E[eV]
+    The photon energy at 1 μm wavelength,
+    the wavelength of near infrared radiation, is approximately 1.2398 eV.
+
+    Usage:
+
+    .. code-block:: python
+
+        photon_to_wavelength(8)
+
+    Parameters
+    ----------
+    energy : number
+        Photon energy in electronvolts
+
+    Returns
+    -------
+    Wavelength in micrometers
+    '''
+    return WAVELENGTH_PHOTON / energy
+
+
+def gaussian_sigma_to_fwhm(sigma):
+    '''
+    Converting between FWHM and sigma of a Gaussian function
+    FWHM = 2.35482004503 * sigma
+
+    https://brainder.org/2011/08/20/gaussian-kernels-convert-fwhm-to-sigma/
+
+    Usage:
+
+    .. code-block:: python
+
+        gaussian_sigma_to_fwhm(0.3)
+
+
+    Parameters
+    ----------
+    sigma : `float`
+
+    Returns
+    -------
+    FWHM -  Full Width at the Half Maximum
+    '''
+    return FWHM_SIGMA_CONVERSION * sigma
+
+
+def gaussian_fwhm_to_sigma(fwhm):
+    '''
+    Converting between FWHM and sigma of a Gaussian function
+    sigma = FWHM / 2.35482004503
+
+    https://brainder.org/2011/08/20/gaussian-kernels-convert-fwhm-to-sigma/
+
+    Usage:
+
+    .. code-block:: python
+
+        gaussian_fwhm_to_sigma(0.023)
+
+    Parameters
+    ----------
+    fwhm : `float`
```

Comment:
```suggestion
    fwhm : float
```

More details in the style guide - it'd be good to become more familiarized with it:
https://pcdshub.github.io/style.html

Reply:
thank you - yes I should